docs/concepts/what-is-a-schedule.md

```diff
@@ -0,0 +1,197 @@
+The following options are available:
+
+- `Skip`: **Default**.
+  Nothing happens – the Workflow Execution is not started.
+- `BufferOne`: Starts the Workflow Execution as soon as the current one completes. The buffer is limited to one.
+  If another Workflow Execution is supposed to start, but there is already one in the buffer, only the one in the buffer eventually starts.
+- `BufferAll`: Allows an unlimited number of Workflows to buffer.
+  They are started sequentially.
+- `CancelOther`: Cancels the running Workflow Execution, and then starts the new one after the old one completes cancellation.
+- `TerminateOther`: Terminates the running Workflow Execution and starts the new one immediately.
+- `AllowAll` Starts any number of concurrent Workflow Executions.
+  With this policy (and only this policy), there may be more than one Workflow Execution, started by the Schedule, running simultaneously.
+
+#### Catchup Window
+
+It may happen that the Temporal Cluster itself is down or unavailable at the time when a Schedule should take an Action.
+When it comes back up, the Catchup Window controls which missed actions should be taken at that point.
+The default is one minute, which means that the Schedule will attempt to take any actions that wouldn't be more than one minute late.
+An outage that lasts longer than the catchup window could lead to missed actions (But you can always Backfill).
+
+#### Pause-on-failure
+
+If this policy is set, a Workflow Execution started by a Schedule that ends with a failure or timeout (but not Cancellation or Termination) will cause the Schedule to automatically pause.
+
+Note that with the `AllowAll` overlap policy, this pause might not apply to the next Run, because the next run might have started before the failed run finished.
+It applies only to Runs that were Scheduled to start after the failed Run finished.
+
+### Last completion result
+
+A Workflow started by a Schedule can obtain the completion result from the most recent successful run. (How you do this depends on which SDK you're using.)
+
+For overlap policies that don't allow overlap, “the most recent successful run” is straightforward to define.
+For the `AllowAll` policy, it refers to the run that completed most recently, at the time that the run in question is started.
+That is, given the overlapping runs below:
+
+```
+time -------------------------------------------->
+ A     |----------------------|
+ B               |-------|
+ C                          |---------------|
+ D                                |--------------T
+```
+
+If D asks for the last completion result at time T, it will get the result of A.
+Not B, even though B started more recently, because A completed later. And not C, even though C completed after A, because the result for D is captured when D is started, not when it's queried.
```

Comment:
```suggestion
If D asks for the last completion result at time T, it gets the result of A.
Not B, even though B started more recently, because A completed later.
And not C, even though C completed after A, because the result for D is captured when D is started, not when it's queried.
```
I love this example! Makes the concept wonderfully concrete.